src/FFmpegReader.cpp

```diff
@@ -1285,9 +1286,18 @@ void FFmpegReader::ProcessVideoPacket(int64_t requested_frame) {
 			}
 
 		} else {
-			// No scaling, use original image size (slower)
-			max_width = info.width;
-			max_height = info.height;
+            // Scale video to equivalent unscaled size
+            // Since the preview window can change sizes, we want to always
+            // scale against the ratio of original video size to timeline size
+            float preview_ratio = 1.0;
+            if (parent->ParentTimeline()) {
+                Timeline *t = (Timeline *) parent->ParentTimeline();
+                preview_ratio = t->preview_width / float(t->info.width);
```

Comment:
Probably missunderstanding this. But I think preview_width is the preview window width, and info.width is the SVG's width.
In that case, the example of a 500px wide image on a 1000px wide preview would have preview_ratio of 2.

Reply:
t->info.width  is  timeline->info.width

Reply:
So, it's `preview window width / timeline width`, which for a 1920px timeline width, and a 800px preview window width, it would be `800 / 1920 = 0.416666667` preview ratio